packages/react/src/components/navigation/sideBar/Item/Item.tsx

```diff
@@ -54,6 +54,14 @@ const ItemWrapper = styled.li`
   }
 `;
 
+const Badge = styled.div`
```

Comment:
I've got the feeling that we can merge this one with the Badge component located here (packages/react/src/components/message/Notification/Badge.tsx). By rewriting it a bit and making the icon as an optional props for example.

WDYT?

Reply:
I thought about that, but it's such a simple component that I'm not entirely sure it's worth it. 
That one is a bit more complex, with a different size too, I feel that adapting it to be usable here would make it even more complex.

Do you feel there's a strong reason to merge them?

Reply:
I thought we used this tiny component anywhere else in the app but I just double checked and I was wrong. So I agree with you, this would be overkill to merge them if we only need this variant here